python project based on machine learning

```diff
@@ -0,0 +1,25 @@
+from texts import text_counter, text training
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.naive_bayes import MultinomialNB
+
+my_string = str(input("enter the string:"))
```

Comment:
This line hangs our continuous integrations tests as discussed in CONTRIBUTING.md